web/src/components/storage/DriveEditor.tsx

```diff
@@ -552,6 +552,8 @@ const PartitionsNoContentSelector = ({ drive, toggleAriaLabel }) => {
 
 const PartitionMenuItem = ({ driveName, mountPath, description }) => {
   const { delete: deletePartition } = usePartition(driveName, mountPath);
+  const volume = useVolume(mountPath);
+  const isRequired = volume.outline?.required || false;
```

Comment:
I'm not against this change, but I was aiming for a simpler work-around only for root ("/") and not for all required file systems.

Using `required` has an interesting implication - all mandatory volumes need to stay together in the same drive. 

This is just a note for the future because with the current product definitions, required is equivalent to only root in all cases.

Reply:
Good point. So, what is better now? I would keep it as it is, and maybe solve the problem in the future with some kind of "move" action. What do you think?

Reply:
Yes. Keep it as it is for now.